src/features/orbitalfeature.jl

```diff
@@ -21,19 +21,19 @@ struct OrbitalFeatureDescriptor <: AbstractEnvironmentFeatureDescriptor
     end
 end
 
-function (ofd::OrbitalFeatureDescriptor)(a::AbstractAtoms)
-    @assert all([el in valenceshell_conf_df[:, :Symbol] for el in elements(a)]) "All elements must be valid and accounted for in the periodic table!"
-    ofd.encoder_decoder(ofd, a)
+function get_value(ofd::OrbitalFeatureDescriptor, el::String)
+    @assert el in valenceshell_conf_df[:, :Symbol] "All elements must be valid and accounted for in the periodic table!"
+
+    getproperty(
+        valenceshell_conf_df[valenceshell_conf_df.Symbol.==el, :][1, :],
+        Symbol("Electronic Structure"),
+    )
```

Comment:
A comment would be nice here so it's immediately clear what's going on. Or should this be a generic one-line function instead? `elementfeature.jl` essentially has the same code and I assume other feature descriptors would also need something similar.